src/components/blog-posts-list/blog-posts-list.js

```diff
@@ -0,0 +1,50 @@
+import { getContentByRoute } from "@greenwood/cli/src/data/queries.js";
+import styles from "./blog-posts-list.module.css";
+
+export default class BlogPostsList extends HTMLElement {
+  async connectedCallback() {
+    const posts = (await getContentByRoute("/blog"))
+      .filter((page) => page.data.published)
+      // we sort in reverse chronologic order, e.g. last in, first out (LIFO)
+      .sort((a, b) =>
+        new Date(a.data.published).getTime() > new Date(b.data.published).getTime() ? -1 : 1,
+      );
+
+    this.innerHTML = `
+      <ul class="${styles.postsList}">
+        ${posts
+          .map((post) => {
+            const { title, route } = post;
+            const { coverImage, abstract = "" } = post.data;
+            const coverBackground = coverImage ? coverImage : "/assets/greenwood-logo-leaf.svg";
+            const coverBackgroundPadding =
+              coverImage && coverImage !== "/assets/greenwood-logo-g.svg" ? "4px" : "14px";
+
+            return `
+              <li class="${styles.postsListItem}">
```

Comment:
@aholtzman observed that the designs promote a more tradition card layout, where in the whole card should probably be wrapped in a link / clickable, and then we should match the heading and body text color / etc accordingly to match
![Screenshot 2024-08-28 at 7 39 23 PM](https://github.com/user-attachments/assets/a2c7dbfa-ca30-4080-971c-87b304c66a01)

And we should also try and keep the rounded corners to similar ratios / degrees, so rounding off the image instead of a circle (and then play around with the padding / spacing a bit)
![Screenshot 2024-08-28 at 7 43 49 PM](https://github.com/user-attachments/assets/fe5a9885-cd87-4506-81e5-4593f563e2c9)